google/mozc
# Add "改竄（カイソ）" to reading_correction.tsv

## Description
This PR adds "改竄（xカイソ, oカイザン）" to `reading_correction.tsv`

## Issue IDs
Issue and/or discussion IDs related to this pull request.
(N/A)

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: [e.g. Windows 11, macOS 13.1, etc]
 - Steps:
   1. type "かいそ"
   2. confirm there is `もしかして: 改竄` in the candidate list

## Additional context
This word is causing a buzz on Twitter with very negative mentions. Adding "もしかして" might make the behavior more acceptable.

However, there could be a downside to this solution: words like ‘開祖’ and ‘海鼠’ are acceptable candidates for ‘かいそ,’ and the ‘もしかして’ candidate might hide these words. In that case, another possible solution would be to simply remove the word.
